src/main/java/uk/gov/hmcts/cmc/claimstore/services/CountyCourtJudgmentService.java

```diff
@@ -78,6 +84,15 @@ public Claim save(final long submitterId, final CountyCourtJudgment countyCourtJ
         return claimWithCCJ;
     }
 
+    private boolean isCountyCourtJudgmentInputValid(CountyCourtJudgment ccj) {
+        return ((ccj.getPaymentOption().equals(PaymentOption.IMMEDIATELY)
```

Comment:
can you split this out either into lines or multiple statements?
It's hard to read